fury-on-minikube/README.md

```diff
@@ -82,7 +84,7 @@ spec:
       networking:
         type: none
       ingress:
-        baseDomain: internal.demo.example.dev
+        baseDomain: demo.example.internal
```

Comment:
We should also edit the references to the base domain in lines 196-198, 214, 223 and 231

Reply:
You're right! I missed them.